### ISSUE TITLE
Cannot run SwiftRewriter

### ISSUE THREAD

**Issue by marcosrocha85** [LuizZak/SwiftRewriter#34]
I'm having an error while running SwiftRewriter.
![Swift Rewriter Error](https://user-images.githubusercontent.com/7257227/71749055-279d3980-2e53-11ea-9d72-34f1643db25b.png)

Branch used: master
Swift used: Swift 5.1.3
Running command: swift run SwiftRewriter --help

**Comment by LuizZak** [comment]
Hmm looks like it's not understanding the omitted `return` keyword, but it should be available in Swift 5.1 onwards. Are you running on Linux by any chance?

**Comment by marcosrocha85** [comment]
No I'm on High Sierra. My Swift is the 5.1.3 ToolChain. I'm using this mac to convert a legacy Objective-C app to Swift. Please don't tell me it won't work on my computer. 😱

**Comment by LuizZak** [comment]
Hmm what's the output of `swift --version`?

**Comment by marcosrocha85** [comment]
```
marcos:~ marcos$ swift --version
Apple Swift version 5.0 (swift-5.0-RELEASE)
Target: x86_64-apple-darwin17.7.0
```

**Comment by LuizZak** [comment]
Hmm as I suspected seems like your CLI is running a different version of Swift. You mentioned you're running a 5.1.3 toolchain, can you try [this guide](https://medium.com/xcblog/switching-swift-versions-inside-xcode-using-toolchains-755b28831c43) and see if you can actually run that from the terminal? That should solve your issues!

**Comment by marcosrocha85** [comment]
Ok, I used [swiftenv](https://github.com/kylef/swiftenv) to change Toolchain and select Swift 5.1.3. But now I'm getting:
```
SwiftRewriter: error: manifest parse error(s):
<unknown>:0: error: Swift does not support the SDK 'MacOSX10.14.sdk'
```
I looked around Google to find how can I change MacOSX10.14.sdk to MacOSX10.15.sdk with Swift 5.1.3 but no success. Do you have any hint?

**Comment by LuizZak** [comment]
Googling around, perhaps this could fix the issue?: https://github.com/tensorflow/swift/issues/208

**Comment by marcosrocha85** [comment]
Yes, I tried that already. I followed [this](https://www.swiftprogrammer.info/swift5_high_sierra.html) tutorial to make Swift 5 running on High Sierra with a difference of instead using XCode 11.1 I downloaded the 11.3 one. I added on my current xcode installation the content of MacOSX.sdk which comes on 11.3 and renamed it to MacOSX10.15.sdk as tutorial asks to. I'm sure my swift is 5.1.3.Toolchain because of console exit from swift --version command. But I don't know how to force it to use sdk 10.15.
